packages/node/src/behavior/state/managed/values/ValuePatcher.ts

```diff
@@ -73,6 +73,9 @@ function StructPatcher(schema: ValueModel, supervisor: RootSupervisor): ValueSup
     // An object mapping name to true iff member is an array
     const memberArrays = {} as Record<string, boolean>;
 
+    // An object mapping alt keys (numeric IDs) to member names, if numeric IDs are defined
+    const memberAltKeys = {} as Record<string, string>;
```

Comment:
Is there any reason to store the indirection from id‐>name->patcher vs just storing id->patcher in memberPatchers?